### PR TITLE
Added radius algorithm

### PR DESCRIPTION
**Pull request by nmansard**

### CODE REVIEW

**Comment by nmansard** [comment]
@fvalenza Could you please add a unittest to validate the values computed by the algorithm against hpp::model? And merge.

File: src/algorithm/geometry.hxx [hunk 1]
```diff
@@ -159,7 +97,53 @@ namespace se3
...
     computeDistances(data_geom);
   }
-  
+
+  /// Given p1..3 being either "min" or "max", return one of the corners of the 
+  /// AABB cub of the FCL object.
+#define SE3_GEOM_AABB(FCL,p1,p2,p3)                                     \
+  SE3::Vector3(                                                         \
+    FCL->aabb_local.p1##_ [0],                                          \
+    FCL->aabb_local.p1##_ [1],                                          \
```
Comment (fvalenza) [inline src/algorithm/geometry.hxx]: Typo error : p1, p2, p3 instead of p1, p1, p1

**Comment by fvalenza** [comment]
In src/benchmark/timings-geometry.hpp there is also an `#include "pinocchio/algorithm/collisions.hpp" that must be replaced by `#include "pinocchio/algorithm/geometry.hpp"`

**Comment by nmansard** [comment]
Corrected, thanks!

**Pull request merged**